Item/Item.module.less

```diff
@@ -105,43 +107,95 @@
 		&.inline {
 			// overriding ui/Item's inline-block
 			display: inline-flex;
+			min-width: @agate-item-inline-min-width;
 			max-width: @agate-item-inline-max-width;
 
 			.itemContent {
 				max-width: @agate-item-inline-content-max-width;
 			}
 
+			&:not(.centered) {
+				.itemContent {
+					min-width: @agate-item-inline-notcentered-itemcontent-min-width;
+				}
+			}
+
 			.itemContent,
 			.content {
 				flex-shrink: 1;
-				overflow: auto;
-			}
-
-			&.hasLabel {
-				.slotAfter {
-					.padding-start-end(@agate-item-slotafter-padding-left, 0);
-				}
+				overflow: hidden;
 			}
 		}
 
 		&.centered {
+			// override Marquee's alignment.
+			.marquee {
+				text-align: center;
+			}
+
 			.itemContent {
-				max-width: fit-content;
+				max-width: 100%;
```

Comment:
Would you please why you had to change this? This made the marquee's width to be 100% so that this whole "override Marquee alignment" happened... ;(

Reply:
I had to made this change because there is a new specification for centered Item as seen below:
![centered-gui](https://user-images.githubusercontent.com/63335823/113125014-d3803100-921e-11eb-9a85-3edca4a718b9.PNG)

If we keep `max-width: fit-content`, when Item is centered, the icons would not position at the start for `labelBefore` and at the end for `labelAfter`. This is a screenshot of centered Item with `max-width: fit-content`.
![fit-content](https://user-images.githubusercontent.com/63335823/113125333-29ed6f80-921f-11eb-960e-593f6a9202c3.PNG)

Reply:
I see. I made some changes based on yours while keeping `max-width: fit-content` here.
You can check it out from https://github.com/enactjs/agate/compare/feature/PLAT-137544...feature/PLAT-137544-mk?expand=1.
Please take a look and you're welcome to review/apply/merge on your branch.

Reply:
Your solution works fine, thanks. Small adjustments had to be made to fix some bugs on Gallium skin for inline Item with slatBefore and slotAfter.